examples/package.json

```diff
@@ -15,10 +15,11 @@
     "@react-page/plugins-video": "0.0.0",
     "@react-page/renderer": "0.0.0",
     "@react-page/ui": "0.0.0",
-    "react": "^16.6.3",
-    "react-dom": "^16.6.3",
-    "@material-ui/core": "3.6.1",
-    "@material-ui/icons": "3.0.1",
+    "react": "link:../node_modules/react",
+    "react-dom": "link:../node_modules/react-dom",
```

Comment:
so this will use the version from the parent right? 

In the parent, react is just a dev-dependency, while for the example it is a prod dependency. I would not mix that. This could break if the example gets autodeployed somewhere using just production node_modules.

Reply:
I didn't get the examples to work without this linking, caused of two differenct react instances. This was the only way I got it to work on my local machine. If you have any other ideas, how to prevent to get two react instances, let me know.

Reply:
maybe because of a version missmatch in package-lock.json or yarn.lock?

Reply:
I don't get `npm install` to run through. So I can't update the package-lock.json of examples. Maybe because I'm running a windows system. With yarn is everything running fine. I also was able to remove the react links, don't know why this wasn't possible last week. 

I'm hoping that one of my colleges can run the `npm install` on his machine and commit the updated lock file to the fork.